app/src/main/java/com/sunasterisk/fooddaily/ui/fragment/home/HomeFragment.kt

```diff
@@ -146,13 +157,15 @@ class HomeFragment:
     }
 
     companion object {
+        private const val REQUEST_FOOD_DETAIL_ACTIVITY = 0
+        private const val ARGUMENT_FOOD_LIST = "ARGUMENT_FOOD_LIST"
         fun newInstance(foods: List<FoodDetail>): HomeFragment = HomeFragment().apply {
-                arguments = bundleOf().apply {
-                    putParcelableArrayList(
-                        Constants.EXTRA_FOOD_LIST,
-                        foods as ArrayList<out Parcelable>
-                    )
-                }
+            arguments = bundleOf().apply {
+                putParcelableArrayList(
+                    ARGUMENT_FOOD_LIST,
+                    foods as ArrayList<out Parcelable>
+                )
```

Comment:
viết thế này có được không 
```suggestion
            arguments = bundleOf(ARGUMENT_FOOD_LIST to (foods as ArrayList<out Parcelable>))
```

Reply:
à đúng rồi, nó giống với truyền data bình thường như mọi lần :3